client/src/components/home/index.jsx

```diff
@@ -192,7 +206,8 @@ const Home = ({ swaps, updateSwaps, clients, swapPairs, update, setupEth, setupT
 
       bal = swapStat.balances[currentSwap.asset]
         .div(10 ** swapPairs[currentSwap.pair][currentSwap.asset].decimals)
-        .toFixed(2)
+        .toFixed(2);
```

Comment:
this should use token decimals, 6 (rather than the full 8) for BTC, 4 (rather than 18) for ETH. 0.0001BTC is 1.32XTZ right now